### PR TITLE
Make README match current best practice from reload.dk

### PR DESCRIPTION
**Pull request by arnested**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,26 +79,33 @@ should reside in any repo where you want to sync security alerts with Jira.
...
           SYSTEM_STATUS_KEY: ${{ secrets.SystemStatusKey }}
           JIRA_TOKEN: ${{ secrets.JiraApiToken }}
           JIRA_HOST: https://reload.atlassian.net
-          JIRA_USER: someuser@reload.dk
+          JIRA_USER: someone@reload.dk
           JIRA_PROJECT: ABC
           JIRA_ISSUE_TYPE: Security
-          JIRA_WATCHERS: someuser@reload.dk,someotheruser@reload.dk
+          JIRA_WATCHERS: customer@example.com,boss@example.com
+          DRY_RUN: ${{ github.event.inputs.dry_run || '0' }}
```
Comment (kasperg) [inline README.md:110]: :+1:

File: README.md [hunk 2]
```diff
@@ -79,26 +79,33 @@ should reside in any repo where you want to sync security alerts with Jira.
 Here is an example setup which runs this action every 6 hours.
```
Comment (kasperg) [inline README.md:79]: If we add the `workflow_dispatch ` part then we should probably also mention it here and perhaps why it is nice to include.

File: README.md [hunk 3]
```diff
@@ -79,26 +79,33 @@ should reside in any repo where you want to sync security alerts with Jira.
...
 jobs:
   syncDrupalSecurityUpdates:
     runs-on: ubuntu-latest
     steps:
       - name: "Sync JIRA security issues from Drupal security updates"
-        uses: reload/github-security-jira
+        uses: reload/drupal-security-jira@main
         env:
-          DRUPAL_HOST: reload.dk
+          DRUPAL_HOST: example.com
```
Comment (kasperg) [inline README.md:101]: :+1:

**Review by kasperg — APPROVED** [review]
:+1: LGTM

File: README.md (hunk 2 above)
Comment (arnested) [reply README.md]: Good point. I have added some text.

**Pull request merged**